interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
We debated about this topic during the call, especially the suggestion to call them “named nodes” rather than “IRIs”. However, [“named node” not used in any of the core RDF specifications](https://www.google.com/search?q=site:www.w3.org%2FTR%2F+%22RDF%22+%22named+node%22), so it might not be clear/intuitive.

That said, I do see the argument that an IRI is a string, and thus not a node; the confusion is especially apparent in the method signature `IRI iri(String iri)`. Maybe we should indeed call them `NamedNode` and create them with `.namedNode`. @bergos, @l00mi, @nicola?

Note that properties are as easy to add as methods in JavaScript; they have the exact same flexibility thanks to `Object.defineProperty`.